readme.md

```diff
@@ -29,7 +29,7 @@ import * as colors from 'yoctocolors';
 console.log(colors.red('Yo!'));
 ```
 
-*This package does not do color support detection. Check out [`supports-color`](https://github.com/chalk/supports-color) if you need that. Node.js will hopefully get [built-in support](https://github.com/nodejs/node/pull/40240) for color support detection at some point.*
+*This package supports basic color detection. Node.js will hopefully get [better support](https://github.com/nodejs/node/pull/40240) for color support detection at some point.*
```

Comment:
I would drop "ode.js will hopefully get [better support](https://github.com/nodejs/node/pull/40240) for color support detection at some point." and just add a TODO to use the better method when it's added to Node.js.